Orange/projection/freeviz.py

```diff
@@ -0,0 +1,348 @@
+
+import numpy
```

Comment:
Sure, thanks. I always do replace `numpy` with `np`.  Mostly before changing anything else. However this file is just a copy from **Prototypes**.